internal/pkg/users/users.go

```diff
@@ -24,7 +24,7 @@ import (
 	"strings"
 )
 
-// GetUID returns uid of given username and logs a warning if its missing
+// GetUID returns uid of given username and returns a warning if its missing
```

Comment:
Does this refer to the second error return value? Maybe better to write "returns an error" instead?

Reply:
You're right, it's not a warning, it's an error.